src/components/Icon/Icon.tsx

```diff
@@ -207,25 +207,39 @@ function Icon({
     backdrop && styles.hasBackdrop,
   );
 
+  const defaultIconProps = {
+    className: styles.Svg,
+    viewBox: '0 0 20 20',
+    focusable: 'false',
+    'aria-hidden': 'true',
+  };
+
   let contentMarkup: React.ReactNode;
   if (source === 'placeholder') {
     contentMarkup = <div className={styles.Placeholder} />;
-  } else if (React.isValidElement(source)) {
-    contentMarkup = source;
+  } else if (typeof source === 'function') {
+    const SourceComponent = source;
+    contentMarkup = <SourceComponent {...defaultIconProps} />;
+  } else if (typeof source === 'string' && isBundledIcon(source)) {
+    // const SourceComponent = BUNDLED_ICONS[source];
+    // contentMarkup = <SourceComponent {...defaultIconProps} />;
```

Comment:
This is the SVGR implementation, below is the old one. I need to hold on the new implementation until we've shipped SVGR in `polaris-icons`, so for now I commented this out and left the old implementation below for ease of testing